builders/interact/ethereum-api/wallets/subwallet.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How to Connect Subwallet to Tanssi
```

Comment:
```suggestion
title: How to Connect SubWallet to Tanssi
```

Looks like Wallet is capitalized on their website, so please update all use cases to be SubWallet